test/Microsoft.Azure.Devices.Edge.Test/Module.cs

```diff
@@ -12,6 +12,35 @@ namespace Microsoft.Azure.Devices.Edge.Test
 
     public class Module : ManualProvisioningFixture
     {
+        private sealed class TempSensorModule
+        {
+            public string Name { get; }
+            public string Image { get; }
+
+            private const string DefaultSensorImage = "mcr.microsoft.com/azureiotedge-simulated-temperature-sensor:1.0";
+            // Make sure it's thread-safe
+            private static readonly object padlock = new object();
```

Comment:
I wouldn't introduce a lock for thread safety unless there's a demonstrated need for it.